articles/quickstart/webapp/express/01-login.md

```diff
@@ -14,30 +14,73 @@ useCase: quickstart
 github:
   path: 01-Login
 ---
+<!-- markdownlint-disable MD002 MD041 -->
 
-<%= include('../../../_includes/_new_app', { hideDownloadSample: true, hideDashboardScreenshot: true }) %>
+## Configure Auth0
 
-<%= include('../../../_includes/_callback_url') %>
+You will need to register your application with Auth0 in order to start authenticating users. Go to the [Applications](${manage_url}/#/applications) screen in the Auth0 dashboard, create a new **Regular Web Application**, and follow the steps below.
 
-:::note
-If you are running your project on `localhost:3000`, your application's callback URL would be `http://localhost:3000/callback`
-:::
+### 1. Configure Callback URL
 
-<%= include('../../../_includes/_logout_url' }) %>
+A callback URL is an application route where Auth0 redirects users after they have authenticated. This URL must be registered with Auth0 or else users will be unable to log in to the application and will get a "Callback URL mismatch" error.
 
-:::note
-If you are running your project on `localhost:3000`, your application's logout URL would be `http://localhost:3000`
-:::
+The callback URL for the application created in this quickstart is `https://localhost:3000/callback`. Paste that in the **Allowed Callback URLs** field for the application you just created.
+
+### 2. Configure Logout URL
+
+A logout URL is an application route that Auth0 can return users to after logging out. This URL must be registered with Auth0 or else users will be unable to log out of the application and will get a "misconfiguration" error.
+
+The logout URL for the application created in this quickstart is `https://localhost:3000`. Paste that in the **Allowed Logout URLs** field for the application you just created, then scroll down and click **Save Changes**.
+
+### 3. Get Your Application Keys
+
+Finally, copy the following fields for your application for use in step 7:
+
+* **Domain**
+* **Client ID**
+
+## Secure Local Server
+
+Applications that handle sensitive data must be served over secure channels. This includes local applications as they may handle the same sensitive data and should be built as close to production-ready as possible.
+
+Complete instructions are below or see our [Secure Local Development guide](/libraries/secure-local-development) for additional explanation.
+
+### 4. Generate Certificates
+
+If you do not already have a method of generating local certificates, [install `mkcert`](https://github.com/FiloSottile/mkcert#installation) and run the following commands in your terminal to generate a certificate and key for this application:
+
+```bash
+#!/bin/sh
+❯ cd /path/to/application/root
+❯ mkcert -install
+❯ mkcert localhost
+```
+
+### 5. Install Certificates
```

Comment:
Here https://github.com/auth0/docs/pull/8668/files#diff-c4d25e0f70b3c3c4e685cd9645275de7R60

We say `Serve the SSL certificate`, what do you think about having consistent verbiage here?